pkg/agent/config.go

```diff
@@ -165,4 +165,14 @@ type Config struct {
 	PCAFilters string `env:"PCA_FILTER"`
 	// PCAServerPort is the port PCA Server starts at, when ENABLE_PCA variable is set to true.
 	PCAServerPort int `env:"PCA_SERVER_PORT" envDefault:"9990"`
+	// MetricsPromoEnable enables prometheus server to collect ebpf agent metrics, default is false.
+	MetricsPromoEnable bool `env:"METRICS_PROMO_ENABLE" envDefault:"false"`
+	// MetricsPromoServerAddress is the address of the prometheus server that collects ebpf agent metrics.
+	MetricsPromoServerAddress string `env:"METRICS_PROMO_SERVER_ADDRESS"`
+	// MetricsPromoPort is the port of the prometheus server that collects ebpf agent metrics.
+	MetricsPromoPort int `env:"METRICS_PROMO_PORT" envDefault:"9090"`
+	// MetricsPrefix is the prefix of the metrics that are sent to prometheus server.
+	MetricsPrefix string `env:"METRICS_PREFIX" envDefault:"ebpf_agent_"`
+	// MetricsNoPanic disables panic on metrics errors, default is false.
+	MetricsNoPanic bool `env:"METRICS_NO_PANIC" envDefault:"false"`
```

Comment:
Maybe we can remove the NoPanic option, not sure if it really brings something useful?
(In FLP we have this "NoPanic" option mostly because the code was initially written with panics and we didn't want that in the operator ... there's not the same background here)